recipes/idaes_minimal/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "idaes_minimal" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IDAES/idaes-pse/archive/bdfddf1d1761bdba408567a83e198d549e8c741d.zip
```

This will make it possible for the update bot to help out:
```suggestion
  url: https://github.com/IDAES/idaes-pse/archive/v{{ version }}.tar.gz
```